cpp/ps0004/150daysprogramming.cpp

```diff
@@ -1,154 +1,62 @@
-// 150daysprogramming.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
 #include <iostream>
-using namespace std;
-#include<string>
-
-//bool isFloat(string num);
-
-bool isFloat(string num) {
-    bool isFloating = false;
-    for (char c : num) {
-        if (c == '.') {
-            isFloating = true;
-            break;
-        }
-    }
-    return isFloating;
-}
-
-
-bool isNumber( string s)
-{
-    for (char c : s) {
-        if (s.length() > 1 && c == '-') {
-            continue;
-        }
-        if (c == '.') {
-            continue;
-        }
-        if (!isdigit(c)) {
-            return false;
-        }
-    }
-    return true;
-}
+#include <string>
 
+using namespace std;
 
+bool isNumber(string s);
+bool isFloat(string num);
 
 int main()
 {
+    // prompting user for number 1.5/1.5
     string input;
     cout << "Enter a number from 32 to 126: ";
     cin >> input;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input